docs/en/tutorials/4_schedule.md

```diff
@@ -48,9 +48,9 @@ lr_config = dict(
     ...)
 ```
 
-Then during training, the program will call [CosineAnealingLrUpdaterHook](https://github.com/open-mmlab/mmcv/blob/f48241a65aebfe07db122e9db320c31b685dc674/mmcv/runner/hooks/lr_updater.py#L227) periodically to update the learning rate.
+Then during training, the program will call [CosineAnealingLrUpdaterHook](https://github.com/open-mmlab/mmcv/blob/0092699fef27a0e6cbe9c37f4c4de2fb6996a1c7/mmcv/runner/hooks/lr_updater.py#L269) periodically to update the learning rate.
```

Comment:
Better change the link to the [stable version](https://github.com/open-mmlab/mmcv/blob/0092699fef27a0e6cbe9c37f4c4de2fb6996a1c7/mmcv/runner/hooks/lr_updater.py#L256)

Reply:
Thanks, your provided link is the same as I provided one (blob/0092....)

Reply:
Reference to code/line must be a blob link, while ref to file should be stable link.

For example, 
1. ref to line: `stable_link/lr_updater.py#L269` is dangerous as the file is changing, L269 may ref to the wrong code.
2. ref to file: `stable_link/lr_updater.py` is the file. When file deleted the link directs to 404, which won't cause misunderstanding.